ed448-goldilocks/src/decaf/points.rs

```diff
@@ -226,7 +226,7 @@ impl CofactorGroup for DecafPoint {
     }
 
     fn is_torsion_free(&self) -> Choice {
-        (self * DECAF_BASEPOINT_ORDER).ct_eq(&Self::IDENTITY)
+        self.ct_eq(&Self::IDENTITY)
     }
```

Comment:
Shouldn't this always be true because Decaf is a prime order group?

Reply:
Yeah, that's what I wrote in the git commit message as well ...
Fixed it.